src/components/Form/style.css

```diff
@@ -0,0 +1,44 @@
+input {
+    border-radius: 25px;
+    height: 47px;
+    border-style: none;
+}
+h1{
+    color: #0D1C43;
+}
+label{
+    color: #737182;
+    font-weight: 700;
+    font-size: 15px;
+}
+p{
+    color: #737182;
+}
+
+.submit_button{
+    background-color: #0D1C43;
+    color: #FFFFFF;
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 20px;
+    width:30%;
+    align-self: center;
```

Comment:
The only thing i could see was maybe adding 

`cursor: pointer;` 
into the `.submit_button` CSS class but other than that it looks super!